packages/fxa-auth-server/lib/senders/emails/partials/appBadges/index.scss

```diff
@@ -3,9 +3,13 @@
  * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
 
 @use '../../global.scss';
-@use '../../layouts/fxa/index.scss';
 
 .app-badges {
+  max-width: 310px !important;
+  text-align: center !important;
+}
+
+.app-badge {
```

Comment:
Hmm, since the text on Subplat is larger, it looks to me like we might want a little bit more space underneath these to give some breathing room/clickable space between the badges and the "contact us" link:

<img width="674" alt="image" src="https://user-images.githubusercontent.com/13018240/147015729-8e7ff834-15a9-4b11-b2da-6c12181340f2.png">


Though, that might require some style shuffling around in layouts, or adding a conditional class here or on `subscriptionSupport`. This is definitely not a blocker.